jgrapht-core/src/test/java/org/jgrapht/traverse/EdgeSelectionTest.java

```diff
@@ -0,0 +1,130 @@
+package org.jgrapht.traverse;
```

Comment:
Could you add the standard header for this file?  Checkstyle enforces this for src files, but we currently have a configuration problem which causes the check to be skipped for test files:

https://github.com/jgrapht/jgrapht/issues/823